### PR TITLE
Create view to create new store

### PR DESCRIPTION
**Pull request by d1eg0dev**
Co-authored-by @EdwardMD <Edwardmi444@gmail.com>
Co-authored-by @ever864 <ever864@gmail.com>
Co-authored-by @carlos-ssh <carlos.anezeh@gmail.com>

### CODE REVIEW

**Review by ever864 — DISMISSED** [review]
Buen trabajo en la creacion del formulario para la creacion del Store!

File: .env [hunk 1]
```diff
@@ -1,2 +1,2 @@
-PG_USERNAME=efpl
```
Comment (HikingCarrot7) [inline .env:1]: Add .env to .gitignore

**Review by EdwardMD — APPROVED** [review]
Nice job!, another great advance!

**Review by BillowyBead34 — APPROVED** [review]
LGMT!

File: .env (hunk 1 above)
Comment (d1eg0dev) [reply .env:1]: Fixed

**Review by BillowyBead34 — APPROVED** [review]
LGTM!

**Review by ever864 — APPROVED** [review]
Buen aporte

**Review by EdwardMD — APPROVED** [review]
good correction

**Review by HikingCarrot7 — APPROVED** [review]
LGTM!

**Pull request merged**